src/plugins/intel_gna/frontend/quantization.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cmath>
+#include <cstdio>
+#include <cstdlib>
+#include <vector>
+#include <cstdint>
+#include "backend/gna_types.h"
+
+#define MAX_OUT_MULTIPLIER 230
+#define MAX_VAL_1B_WEIGHT 127
+#define MAX_VAL_1B_FEAT 64
+#define MAX_VAL_1B_BIAS 127
+#define MAX_VAL_2B_WEIGHT 16384
+#define MAX_VAL_2B_FEAT 16384
+#define MAX_VAL_4B_BIAS 1073741824
+
+struct QuantizationCallbackData {
+    const size_t num_rows;
+    const size_t num_columns;
+    const size_t fq_levels;
+    const size_t fq_num_stats;
+    const float* fq_ptr_input_low;
+    const float* fq_ptr_input_high;
+    const float* fq_ptr_output_low;
+    const float* fq_ptr_output_high;
```

Comment:
Do we really need pointers here ?
couldn't we use float values of even std::vector<float>?

Reply:
This wasn't considered as part of the changes, but I'll take a look.